test/e2e/proxy_test.go

```diff
@@ -71,3 +71,29 @@ func TestSnowKubernetes121UbuntuProxyConfig(t *testing.T) {
 	)
 	runProxyConfigFlow(test)
 }
+
+func TestTinkerbellKubernetes122UbuntuProxyConfig(t *testing.T) {
+	test := framework.NewClusterE2ETest(
+		t,
+		framework.NewVSphere(t, framework.WithUbuntu122(),
+			framework.WithPrivateNetwork()),
+		framework.WithClusterFiller(api.WithControlPlaneCount(1)),
+		framework.WithClusterFiller(api.WithWorkerNodeCount(1)),
+		framework.WithClusterFiller(api.WithKubernetesVersion(v1alpha1.Kube122)),
+		framework.WithProxy(framework.VsphereProxyRequiredEnvVars),
+	)
+	runProxyConfigFlow(test)
+}
```

Comment:
Why did you add this test? It looks like this is going to create a vSphere cluster with proxy config?

Reply:
Possibly to debug cli-tools container helm issue with proxy, lol